src/System.Net.Requests/pkg/System.Net.Requests.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Net.Requests.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Well, in that case, why has System.Net.Requests 4.1 not being supported on netcore50 (UWP)?  This library package is "cross-platform", i.e. across all Windows (UWP and non-UWP) as well as *Nix.  We should make sure that we enable 4.1 for UWP and my original comment seems to suggest that we are not right now.

Or am I mistaken?